performance/example_usage.cc

```diff
@@ -0,0 +1,193 @@
+#include <iostream>
```

Comment:
Is this file only used for generating docs? I don't think it's actually compiled anywhere, is it?

Reply:
The file is called example_usage, so I think it's just to show how to use it.